### PR TITLE
Move Infinity rep mochitest to jest

### PR DESCRIPTION
**Pull request by sidthesloth92**
I've commit the code for infinity to jest migration

### CODE REVIEW

**Comment by sidthesloth92** [comment]
@nchevobbe Have migrated infinity to jest.

File: packages/devtools-reps/src/reps/tests/infinity.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    expect(getRep(stub)).toBe(InfinityRep.rep);
+  });
+  
+  it("Infinity rep has expected text content for negative Infinity", () => {
+    const renderedComponent = shallow(Rep({
+      object: stub
+    }));
+    expect(renderedComponent.text()).toEqual("-Infinity");
+  });
+});
```
Comment (nchevobbe) [inline packages/devtools-reps/src/reps/tests/infinity.js]: files need to have an empty line at the bottom, let's add it

Comment (sidthesloth92) [inline packages/devtools-reps/src/reps/tests/infinity.js]: Anything I have to do here? Thank you for adding the spaces.

Comment (nchevobbe) [inline packages/devtools-reps/src/reps/tests/infinity.js]: No, I was waiting for CI to validate the PR.
I'll merge it :)

**Comment by jasonLaster** [comment]
Thanks @sidthesloth92! Feel free to join our slack and say hi!

https://devtools-html.slack.com

**Pull request merged**